app/src/main/java/com/eveningoutpost/dexdrip/Models/LibreOOPAlgorithm.java

```diff
@@ -183,277 +184,285 @@ static public  UnlockBuffers sendGetBluetoothEnablePayload(boolean increaseConne
         ReadingData readingData = new ReadingData();
 
         readingData.trend = new ArrayList<GlucoseData>();
-        
-        double factor = 1;
-        if(use_raw) {
-            // When handeling raw, data is expected to be bigger in a factor of 1000 and 
-            // is then devided by Constants.LIBRE_MULTIPLIER
-            factor = 1000 / Constants.LIBRE_MULTIPLIER;
-        }
-        
+
         // Add the first object, that is the current time
         GlucoseData glucoseData = new GlucoseData();
         glucoseData.sensorTime = oOPResults.currentTime;
         glucoseData.realDate = oOPResults.timestamp;
-        glucoseData.glucoseLevel = (int)(oOPResults.currentBg * factor);
-        glucoseData.glucoseLevelRaw = (int)(oOPResults.currentBg * factor);
+        glucoseData.glucoseLevel = (int) (oOPResults.currentBg);
+        glucoseData.glucoseLevelRaw = (int) (oOPResults.currentBg);
 
-        verifyTime( glucoseData.sensorTime, "LibreOOPAlgorithm", null);
+        verifyTime(glucoseData.sensorTime, "LibreOOPAlgorithm", null);
         readingData.trend.add(glucoseData);
-        
+
         // TODO: Add here data of last 10 minutes or whatever.
-        
-        
+
+
         // Add the historic data
         readingData.history = new ArrayList<GlucoseData>();
-        for(HistoricBg historicBg : oOPResults.historicBg) {
-            if(historicBg.quality == 0) {
+        for (HistoricBg historicBg : oOPResults.historicBg) {
+            if (historicBg.quality == 0) {
                 glucoseData = new GlucoseData();
                 glucoseData.realDate = oOPResults.timestamp + (historicBg.time - oOPResults.currentTime) * 60000;
-                glucoseData.glucoseLevel = (int)(historicBg.bg * factor);
-                glucoseData.glucoseLevelRaw = (int)(historicBg.bg * factor);
+                glucoseData.glucoseLevel = (int) (historicBg.bg);
+                glucoseData.glucoseLevelRaw = (int) (historicBg.bg);
                 readingData.history.add(glucoseData);
             }
         }
-        
+
         // Add the current point again. This is needed in order to have the last gaps closed.
         // TODO: Base this on real BG values.
         glucoseData = new GlucoseData();
         glucoseData.realDate = oOPResults.timestamp;
-        glucoseData.glucoseLevel = (int)(oOPResults.currentBg * factor);
-        glucoseData.glucoseLevelRaw = (int)(oOPResults.currentBg * factor);
+        glucoseData.glucoseLevel = (int) (oOPResults.currentBg);
+        glucoseData.glucoseLevelRaw = (int) (oOPResults.currentBg);
         readingData.history.add(glucoseData);
-        
+
         Log.d(TAG, "handleData Created the following object " + readingData.toString());
-        LibreAlarmReceiver.CalculateFromDataTransferObject(readingData, false, use_raw);
+        LibreAlarmReceiver.CalculateFromDataTransferObject(readingData, false, false);
```

Comment:
Is the parameter `use_raw` still in use, elsewhere, if you set it to a constant `false` here? If not, please update the function definition.

Reply:
On line 206 of LibreAlarmReceiver.java in this pr (Sorry, I did not find a way to create a link there) it might be called with true.